### ISSUE TITLE
Insecure evaluations of form conditions

### ISSUE THREAD

**Issue by XaviTorello** [networknt/react-schema-form#90]
To evaluate form conditions is used the `eval` built-in without any kind of filtering or protection:
https://github.com/networknt/react-schema-form/blob/92397dbd4c412c94b252fdb40a84f068204a0f65/src/SchemaForm.js#L51-L53

It can be dangerous if someone is able to modify the condition and tries to inject some code.

Dangerous conditions like `"model = {}"` are executed being able to overriding our app context, or interact with our DOM (i.e. `"alert('ROLF')"`, `"document.write..."`, `"document.cookie..."`, ...)

In summary it is vulnerable to XSS exploiting form conditions.

Please, review it and provide a fix
